diff -urNp -x '*.orig' Tk-804.034.org/Tk/Widget.pm Tk-804.034/Tk/Widget.pm
--- Tk-804.034.org/Tk/Widget.pm	2015-05-22 08:50:55.000000000 +0200
+++ Tk-804.034/Tk/Widget.pm	2021-05-08 23:13:26.906043565 +0200
@@ -60,7 +60,7 @@ sub import
 sub True  { 1 }
 sub False { 0 }
 
-use Tk::Submethods( 'grab' =>  [qw(current status release -global)],
+use Tk::Submethods ( 'grab' =>  [qw(current status release -global)],
                     'focus' => [qw(-force -lastfor)],
                     'pack'  => [qw(configure forget info propagate slaves)],
                     'grid'  => [qw(bbox columnconfigure configure forget info location propagate remove rowconfigure size slaves)],
