--- Tk800.022/Contrib/book-examples/f-15.5.pl.wiget	Tue Jul 27 20:20:04 1999
+++ Tk800.022/Contrib/book-examples/f-15.5.pl	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 #
 # Figure 15.5, p153
--- Tk800.022/Contrib/book-examples/f-16.1.pl.wiget	Tue Jul 27 20:20:04 1999
+++ Tk800.022/Contrib/book-examples/f-16.1.pl	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 #
 # Figure 16.1, p159
--- Tk800.022/Contrib/book-examples/f-16.10.pl.wiget	Tue Jul 27 20:20:04 1999
+++ Tk800.022/Contrib/book-examples/f-16.10.pl	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 #
 # Figure 16.8, p166
--- Tk800.022/Contrib/book-examples/f-16.2.pl.wiget	Tue Jul 27 20:20:04 1999
+++ Tk800.022/Contrib/book-examples/f-16.2.pl	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 #
 # Figure 16.2, p161
--- Tk800.022/Contrib/book-examples/f-16.3.pl.wiget	Tue Jul 27 20:20:04 1999
+++ Tk800.022/Contrib/book-examples/f-16.3.pl	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 #
 # Figure 16.3, p162
--- Tk800.022/Contrib/book-examples/f-16.6.pl.wiget	Tue Jul 27 20:20:04 1999
+++ Tk800.022/Contrib/book-examples/f-16.6.pl	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 #
 # Figure 16.6, p165
--- Tk800.022/Contrib/book-examples/f-16.7.pl.wiget	Tue Jul 27 20:20:04 1999
+++ Tk800.022/Contrib/book-examples/f-16.7.pl	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 #
 # Figure 16.7, p166
--- Tk800.022/Contrib/book-examples/f-16.8.pl.wiget	Tue Jul 27 20:20:04 1999
+++ Tk800.022/Contrib/book-examples/f-16.8.pl	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 #
 # Figure 16.8, p166
--- Tk800.022/Contrib/book-examples/f-16.9.pl.wiget	Tue Jul 27 20:20:04 1999
+++ Tk800.022/Contrib/book-examples/f-16.9.pl	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 #
 # Figure 16.8, p166
--- Tk800.022/Contrib/TickerTape.wiget	Tue Jul 27 20:20:04 1999
+++ Tk800.022/Contrib/TickerTape	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 =head1 NAME 
 
--- Tk800.022/Contrib/fd_demo.wiget	Tue Jul 27 20:20:04 1999
+++ Tk800.022/Contrib/fd_demo	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 require Tk::FileDialog;
--- Tk800.022/Contrib/testlistbox.wiget	Tue Jul 27 20:20:04 1999
+++ Tk800.022/Contrib/testlistbox	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 use Tk;
 use Tk::TiedListbox;
 
--- Tk800.022/DragDrop/drag_test.wiget	Tue Jul 27 20:20:04 1999
+++ Tk800.022/DragDrop/drag_test	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 use strict;
 use Tk::DragDrop qw(Sun);
--- Tk800.022/DragDrop/has_sun.wiget	Tue Jul 27 20:20:04 1999
+++ Tk800.022/DragDrop/has_sun	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 
 my $mw = MainWindow->new;
--- Tk800.022/DragDrop/local_test.wiget	Tue Jul 27 20:20:04 1999
+++ Tk800.022/DragDrop/local_test	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 use strict;
 use Tk::DragDrop;
--- Tk800.022/DragDrop/motion_test.wiget	Tue Jul 27 20:20:04 1999
+++ Tk800.022/DragDrop/motion_test	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 use strict;
 use Tk::DragDrop;
--- Tk800.022/DragDrop/selection.wiget	Tue Jul 27 20:20:04 1999
+++ Tk800.022/DragDrop/selection	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 use strict;
 
--- Tk800.022/DragDrop/site_test.wiget	Sat Aug 21 22:03:26 1999
+++ Tk800.022/DragDrop/site_test	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 use strict;
 use Tk::DropSite;
--- Tk800.022/Extensions/ImageBack/background.wiget	Tue Jul 27 20:20:05 1999
+++ Tk800.022/Extensions/ImageBack/background	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 use Tk::ImageBack;
 use Tk::Photo;
--- Tk800.022/HList/Tix2perl.wiget	Tue Jul 27 20:20:05 1999
+++ Tk800.022/HList/Tix2perl	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk::Pretty;
 
--- Tk800.022/TixGrid/tgdemo_config.wiget	Tue Jul 27 20:20:07 1999
+++ Tk800.022/TixGrid/tgdemo_config	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 # tixgrid demo
 #	standalone or sourceable in ptksh
 
--- Tk800.022/WinPhoto/try.wiget	Tue Jul 27 20:20:11 1999
+++ Tk800.022/WinPhoto/try	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;             
 use Tk::WinPhoto;
 use Cwd;
--- Tk800.022/Xlib/drops.wiget	Tue Jul 27 20:20:11 1999
+++ Tk800.022/Xlib/drops	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 
 use Tk;
--- Tk800.022/Xlib/load.wiget	Tue Jul 27 20:20:11 1999
+++ Tk800.022/Xlib/load	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 use Tk;
 use Tk::Xlib;
--- Tk800.022/Xlib/motif_site.wiget	Tue Jul 27 20:20:11 1999
+++ Tk800.022/Xlib/motif_site	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 use Tk::Xlib;
 use Data::Dumper;
--- Tk800.022/Xlib/tree_demo.wiget	Tue Jul 27 20:20:11 1999
+++ Tk800.022/Xlib/tree_demo	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 use Tk::Xlib;
 use Tk::HList;
--- Tk800.022/bin/defs.wiget	Tue Jul 27 20:20:11 1999
+++ Tk800.022/bin/defs	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl 
+#!/usr/bin/perl 
 
 %def   = ();
 %need  = ();
--- Tk800.022/bin/defunc.wiget	Tue Jul 27 20:20:11 1999
+++ Tk800.022/bin/defunc	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 while (<>)
  {
--- Tk800.022/bin/newstyle.wiget	Tue Jul 27 20:20:11 1999
+++ Tk800.022/bin/newstyle	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 @files = @ARGV;
 
--- Tk800.022/bin/perms.wiget	Tue Jul 27 20:20:11 1999
+++ Tk800.022/bin/perms	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 use File::Find;
 
--- Tk800.022/bin/unres.wiget	Tue Jul 27 20:20:11 1999
+++ Tk800.022/bin/unres	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 %done = ();
 %defs = ();
--- Tk800.022/bin/uselib.wiget	Tue Jul 27 20:20:11 1999
+++ Tk800.022/bin/uselib	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl 
+#!/usr/bin/perl 
 
 use File::Find;
 
--- Tk800.022/compile/findloader.wiget	Tue Jul 27 20:20:11 1999
+++ Tk800.022/compile/findloader	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Config; 
 my $sfx = $Config{'lib_ext'};
 foreach my $dir (@INC)
--- Tk800.022/compile/splitfunc.wiget	Tue Jul 27 20:20:11 1999
+++ Tk800.022/compile/splitfunc	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use IO::File;
 
 while (<>)
--- Tk800.022/demos/browse.wiget	Tue Jul 27 20:20:12 1999
+++ Tk800.022/demos/browse	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 #
 # This script generates a directory browser, which lists the working directory and allows you to open files or subdirectories
 # by double-clicking.
--- Tk800.022/demos/color_editor.wiget	Tue Jul 27 20:20:12 1999
+++ Tk800.022/demos/color_editor	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 #
 # ColorEditor demonstration.
 
--- Tk800.022/demos/composite.wiget	Tue Jul 27 20:20:12 1999
+++ Tk800.022/demos/composite	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 
 use English;
--- Tk800.022/demos/dialog.wiget	Tue Jul 27 20:20:17 1999
+++ Tk800.022/demos/dialog	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 # Demonstrate how to use Dialog to create and manipulate dialog objects.
 # lusol@Lehigh.EDU  95/06/14
--- Tk800.022/demos/hello.wiget	Tue Jul 27 20:20:17 1999
+++ Tk800.022/demos/hello	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 #
 # Simple Tk script to create a button that prints "Hello, world".  Click on 
 # the button to terminate the program.
--- Tk800.022/demos/ixset.wiget	Tue Jul 27 20:20:17 1999
+++ Tk800.022/demos/ixset	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 # A nice interface to "xset" to change X server settings
 #
--- Tk800.022/demos/rmt.wiget	Fri Jul 30 17:56:30 1999
+++ Tk800.022/demos/rmt	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -wT
+#!/usr/bin/perl -wT
 #
 # This script implements a simple remote-control mechanism for Tk 
 # applications.  It allows you to select an application and then type 
--- Tk800.022/demos/rolodex.wiget	Tue Jul 27 20:20:17 1999
+++ Tk800.022/demos/rolodex	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 #
 # This script was written as an entry in Tom LaStrange's rolodex benchmark. 
 # It creates something that has some of the look and feel of a rolodex program,# although it's lifeless and doesn't actually do the rolodex application.
--- Tk800.022/demos/timer.wiget	Tue Jul 27 20:20:17 1999
+++ Tk800.022/demos/timer	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 #
 # This script generates a counter with start and stop buttons.  Exit with 
 # Ctrl/c or Ctrl/q.
--- Tk800.022/demos/tom.wiget	Tue Jul 27 20:20:17 1999
+++ Tk800.022/demos/tom	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 #
 # Sample program to get Tom Christiansen going with Perl/Tk.
 
--- Tk800.022/demos/widget.wiget	Fri Mar 31 14:55:09 2000
+++ Tk800.022/demos/widget	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 require 5.004;
 
--- Tk800.022/demos/widget_old.wiget	Tue Jul 27 20:20:18 1999
+++ Tk800.022/demos/widget_old	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 #
 # This script demonstrates the various widgets provided by Tk, along with many of the features of the Tk toolkit.  This file
 # only contains code to generate the main window for the application, which invokes individual demonstrations.  The code for 
--- Tk800.022/pTk/mTk/tclmerge.wiget	Tue Jul 27 20:20:46 1999
+++ Tk800.022/pTk/mTk/tclmerge	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use strict;
 
--- Tk800.022/pTk/mTk/tixmerge.wiget	Tue Jul 27 20:20:58 1999
+++ Tk800.022/pTk/mTk/tixmerge	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use strict;
 
--- Tk800.022/pTk/LangIO.h.PL.wiget	Tue Jul 27 20:20:25 1999
+++ Tk800.022/pTk/LangIO.h.PL	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Config;
 my @gard = qw(
 STDIO_H
--- Tk800.022/pTk/Tcl-pTk.wiget	Sun Dec 12 14:58:36 1999
+++ Tk800.022/pTk/Tcl-pTk	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Carp;
 
--- Tk800.022/pTk/chext.wiget	Tue Jul 27 20:20:26 1999
+++ Tk800.022/pTk/chext	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 open(NEED,"<need") || die;
 while (<NEED>)
--- Tk800.022/pTk/counts.wiget	Tue Jul 27 20:20:26 1999
+++ Tk800.022/pTk/counts	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 sub count
 {
--- Tk800.022/pTk/findX.wiget	Tue Jul 27 20:20:26 1999
+++ Tk800.022/pTk/findX	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 BEGIN { require Exporter };
 
--- Tk800.022/pTk/findcore.wiget	Tue Jul 27 20:20:26 1999
+++ Tk800.022/pTk/findcore	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 BEGIN { require Exporter };
 
--- Tk800.022/pTk/findstrings.wiget	Tue Jul 27 20:20:26 1999
+++ Tk800.022/pTk/findstrings	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 
 %table = ();
--- Tk800.022/pTk/from.wiget	Tue Jul 27 20:20:26 1999
+++ Tk800.022/pTk/from	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 
 use File::Find;
--- Tk800.022/pTk/genexc.wiget	Tue Jul 27 20:20:26 1999
+++ Tk800.022/pTk/genexc	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 my %mTk;
 my $dir;
--- Tk800.022/pTk/makeenglish.wiget	Tue Jul 27 20:21:09 1999
+++ Tk800.022/pTk/makeenglish	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 # BEGIN { push(@INC, qw(../../../lib ../../lib ../lib lib)) }
 
 die "Must be executed from Tk/pTk distribution directory\n" unless -f "Lang.h";
--- Tk800.022/pTk/mkVFunc.wiget	Sat Nov 20 22:09:59 1999
+++ Tk800.022/pTk/mkVFunc	Wed Nov 15 17:17:16 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;               
 
 my %Ignore;
--- Tk800.022/pTk/mkXinfo.wiget	Tue Jul 27 20:21:09 1999
+++ Tk800.022/pTk/mkXinfo	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 $file = shift;
 
--- Tk800.022/pTk/mkexc.wiget	Tue Jul 27 20:21:09 1999
+++ Tk800.022/pTk/mkexc	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 use vars qw(%ref %def);
 require "process_object";
--- Tk800.022/pTk/mkmethods.wiget	Tue Jul 27 20:21:09 1999
+++ Tk800.022/pTk/mkmethods	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 $dst = shift;
 
--- Tk800.022/pTk/tdiff.wiget	Tue Jul 27 20:21:10 1999
+++ Tk800.022/pTk/tdiff	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 sub fail
 {
--- Tk800.022/pTk/why.wiget	Sat Nov 20 14:55:25 1999
+++ Tk800.022/pTk/why	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use strict;        
 
--- Tk800.022/pod/extract/extpod.wiget	Tue Nov  9 14:18:00 1999
+++ Tk800.022/pod/extract/extpod	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;            
 use Getopt::Std;
 use ExtUtils::Manifest "/mani/";
--- Tk800.022/pod/recat.wiget	Tue Nov  9 14:18:01 1999
+++ Tk800.022/pod/recat	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 my $sec;            
 my %cat;       
--- Tk800.022/pod/subs.wiget	Tue Nov  9 14:18:01 1999
+++ Tk800.022/pod/subs	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use blib;
 use Tk;                     
 use strict;
--- Tk800.022/add_version.wiget	Sat Jan 22 15:18:58 2000
+++ Tk800.022/add_version	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 use File::Find;
 use Getopt::Std;
--- Tk800.022/adj_demo.wiget	Tue Jul 27 20:20:11 1999
+++ Tk800.022/adj_demo	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl  -w
+#!/usr/bin/perl  -w
 use Tk;
 require Tk::TextUndo;
 
--- Tk800.022/after_demo.wiget	Tue Jul 27 20:20:11 1999
+++ Tk800.022/after_demo	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 use strict;
--- Tk800.022/after_leak.wiget	Sun Dec 19 21:49:02 1999
+++ Tk800.022/after_leak	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 use Devel::Leak;
--- Tk800.022/animate.wiget	Tue Jul 27 20:20:11 1999
+++ Tk800.022/animate	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 
 my $mw = MainWindow->new;
--- Tk800.022/autoraise.wiget	Tue Jul 27 20:20:11 1999
+++ Tk800.022/autoraise	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 
--- Tk800.022/basic_demo.wiget	Thu Apr 27 16:38:55 2000
+++ Tk800.022/basic_demo	Wed Nov 15 17:16:24 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 # use Memory (2440);
 
--- Tk800.022/bindtest.wiget	Tue Jul 27 20:20:11 1999
+++ Tk800.022/bindtest	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 use Tk::Menubar;
 
--- Tk800.022/build.wiget	Tue Jul 27 20:20:11 1999
+++ Tk800.022/build	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Config;
 my $make = $Config{'make'};
 my @para;
--- Tk800.022/bulkedit.wiget	Sun Dec 19 22:04:14 1999
+++ Tk800.022/bulkedit	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 use Tk;
 use Tk::LabEntry;
--- Tk800.022/canvas_ps.wiget	Tue Jul 27 20:20:11 1999
+++ Tk800.022/canvas_ps	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 
--- Tk800.022/canvas_scroll.wiget	Thu Nov 25 11:57:29 1999
+++ Tk800.022/canvas_scroll	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 
--- Tk800.022/cbtest.wiget	Tue Jul 27 20:20:11 1999
+++ Tk800.022/cbtest	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 use Tk;
 use Tk::Pretty;
--- Tk800.022/check_syms.wiget	Tue Jul 27 20:20:11 1999
+++ Tk800.022/check_syms	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use File::Find;     
 use Config; 
 my @libs;
--- Tk800.022/checklen.wiget	Tue Jul 27 20:20:11 1999
+++ Tk800.022/checklen	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use ExtUtils::Manifest "/mani/";
 
--- Tk800.022/classtree.wiget	Tue Jul 27 20:20:11 1999
+++ Tk800.022/classtree	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 use Tk;    
 use Tk::widgets qw(Tree Menubar);
--- Tk800.022/cleanup.wiget	Tue Jul 27 20:20:11 1999
+++ Tk800.022/cleanup	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use File::Find;
 find(\&wanted,'.');
 
--- Tk800.022/clip_bug.wiget	Tue Jul 27 20:20:11 1999
+++ Tk800.022/clip_bug	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 # Show a Tk canvas bug:
 # when scrolling a canvas, clipping for embedded windows
 # uses the embedded windows parent, not the canvas.
--- Tk800.022/copydate.wiget	Tue Jul 27 20:20:12 1999
+++ Tk800.022/copydate	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl 
+#!/usr/bin/perl 
 
 use File::Find;
 use Time::Local;
--- Tk800.022/cursor_demo.wiget	Tue Jul 27 20:20:12 1999
+++ Tk800.022/cursor_demo	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 
 my $mw = MainWindow->new;
--- Tk800.022/derived_test.wiget	Tue Jul 27 20:20:18 1999
+++ Tk800.022/derived_test	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 require Tk::Text;
 
--- Tk800.022/destroy_test.wiget	Tue Jul 27 20:20:18 1999
+++ Tk800.022/destroy_test	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 
 use Tk;
--- Tk800.022/dfpy.wiget	Tue Jul 27 20:20:18 1999
+++ Tk800.022/dfpy	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 use ExtUtils::Manifest "/mani/";
 use Cwd;
--- Tk800.022/disbutton.wiget	Tue Jul 27 20:20:18 1999
+++ Tk800.022/disbutton	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 package StickyButton;
 require Tk::Widget;
--- Tk800.022/dnd_demo.wiget	Tue Jul 27 20:20:18 1999
+++ Tk800.022/dnd_demo	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 #
 # Demo code to exhibit features of dynamic construction and destruction of 
 # local (within app.) Drag and Drop sources and sinks. Written for and tested 
--- Tk800.022/dump_test.wiget	Tue Jul 27 20:20:24 1999
+++ Tk800.022/dump_test	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 my $top = MainWindow->new;
 my $i   = $top->Photo(-file => Tk->findINC('icon.gif'));
--- Tk800.022/embed.wiget	Tue Jul 27 20:20:24 1999
+++ Tk800.022/embed	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w 
+#!/usr/bin/perl -w 
 
 use Tk;
 use IO::File;
--- Tk800.022/error_prop.wiget	Tue Jul 27 20:20:24 1999
+++ Tk800.022/error_prop	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 use Tk::ErrorDialog;
--- Tk800.022/ewidth_demo.wiget	Tue Jul 27 20:20:24 1999
+++ Tk800.022/ewidth_demo	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 use strict;
 
--- Tk800.022/findbase.wiget	Tue Jul 27 20:20:24 1999
+++ Tk800.022/findbase	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use File::Find;
 
--- Tk800.022/floppy.wiget	Tue Jul 27 20:20:24 1999
+++ Tk800.022/floppy	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 use ExtUtils::Manifest "/mani/";
 use Cwd;
--- Tk800.022/font_demo.wiget	Sun Dec 19 18:25:43 1999
+++ Tk800.022/font_demo	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 use Tk::X11Font;
 use Tk::ROText;
--- Tk800.022/font_test.wiget	Tue Jul 27 20:20:24 1999
+++ Tk800.022/font_test	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 
 
--- Tk800.022/fonts.wiget	Tue Jul 27 20:20:24 1999
+++ Tk800.022/fonts	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 use Tk;              
 use Data::Dumper;
--- Tk800.022/gdb.plx.wiget	Tue Jul 27 20:20:24 1999
+++ Tk800.022/gdb.plx	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Config;
 open(INIT,">.gdbinit") || die;
 my $perl = $^X;
--- Tk800.022/geo_mgr.wiget	Sat Dec 18 20:52:30 1999
+++ Tk800.022/geo_mgr	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 
--- Tk800.022/geom.wiget	Tue Jul 27 20:20:24 1999
+++ Tk800.022/geom	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 
--- Tk800.022/grepc.wiget	Mon Mar 13 15:24:03 2000
+++ Tk800.022/grepc	Wed Nov 15 17:15:49 2000
@@ -1,4 +1,4 @@
-#!/tools/local/perl -w
+#!/usr/bin/perl -w
 use strict;
 use vars qw($file);
 use Data::Dumper;
--- Tk800.022/grepperl.wiget	Thu Dec 30 15:52:10 1999
+++ Tk800.022/grepperl	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl 
+#!/usr/bin/perl 
 
 use Getopt::Std;
 
--- Tk800.022/grid_adj.wiget	Tue Jul 27 20:20:24 1999
+++ Tk800.022/grid_adj	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl  -w
+#!/usr/bin/perl  -w
 use Tk;
 require Tk::TextUndo;
 require Tk::Adjuster;
--- Tk800.022/grid_test.wiget	Tue Jul 27 20:20:24 1999
+++ Tk800.022/grid_test	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 my $mw = MainWindow->new;
 open(FILE,__FILE__);
--- Tk800.022/gridbug.wiget	Tue Jul 27 20:20:24 1999
+++ Tk800.022/gridbug	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 use Carp;
 $top = MainWindow->new();
--- Tk800.022/hackMM.wiget	Tue Jul 27 20:20:24 1999
+++ Tk800.022/hackMM	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use File::Find;
 
--- Tk800.022/hlfm.wiget	Tue Jul 27 20:20:24 1999
+++ Tk800.022/hlfm	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 use Tk;
 require Tk::Dirlist;
--- Tk800.022/hlist_bg.wiget	Tue Jul 27 20:20:24 1999
+++ Tk800.022/hlist_bg	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 require Tk::HList;
 
--- Tk800.022/iconwin.wiget	Tue Jul 27 20:20:24 1999
+++ Tk800.022/iconwin	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 
--- Tk800.022/image_test.wiget	Tue Jul 27 20:20:24 1999
+++ Tk800.022/image_test	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 
--- Tk800.022/inline_bitmap.wiget	Tue Jul 27 20:20:24 1999
+++ Tk800.022/inline_bitmap	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 use Tk;             
 
--- Tk800.022/keyWords.wiget	Tue Jul 27 20:20:24 1999
+++ Tk800.022/keyWords	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use File::Find;
 
--- Tk800.022/labelpack.wiget	Tue Jul 27 20:20:24 1999
+++ Tk800.022/labelpack	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 require Tk::LabEntry;
--- Tk800.022/lbformat_demo.wiget	Tue Jul 27 20:20:24 1999
+++ Tk800.022/lbformat_demo	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 use Tk;
 use Tk::Font;
--- Tk800.022/leak_test.wiget	Sun Dec 19 21:54:49 1999
+++ Tk800.022/leak_test	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 use Devel::Leak;
--- Tk800.022/lentry.wiget	Tue Jul 27 20:20:24 1999
+++ Tk800.022/lentry	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 
--- Tk800.022/local_demo.wiget	Tue Jul 27 20:20:24 1999
+++ Tk800.022/local_demo	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 use Tk qw(MainLoop *widget);
 use Carp;
--- Tk800.022/ls_table.wiget	Tue Jul 27 20:20:24 1999
+++ Tk800.022/ls_table	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 use Tk::Table;
--- Tk800.022/lsfonts.wiget	Tue Jul 27 20:20:24 1999
+++ Tk800.022/lsfonts	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 use Tk::Xlib;
--- Tk800.022/menubar.wiget	Tue Jul 27 20:20:24 1999
+++ Tk800.022/menubar	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 
--- Tk800.022/menubug.wiget	Tue Jul 27 20:20:24 1999
+++ Tk800.022/menubug	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 use strict;
 
--- Tk800.022/mitem_test.wiget	Tue Jul 27 20:20:24 1999
+++ Tk800.022/mitem_test	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
  
 require 5.003;
 use English;
--- Tk800.022/mkExt.wiget	Tue Jul 27 20:20:24 1999
+++ Tk800.022/mkExt	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 sub translate
 {
--- Tk800.022/mkdist.wiget	Tue Jul 27 20:20:24 1999
+++ Tk800.022/mkdist	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 
 use ExtUtils::Manifest "/mani/";
--- Tk800.022/motif_hack.wiget	Tue Jul 27 20:20:24 1999
+++ Tk800.022/motif_hack	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Data::Dumper;
 use strict;
 use Tk;
--- Tk800.022/nested_dialog.wiget	Tue Jul 27 20:20:24 1999
+++ Tk800.022/nested_dialog	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 use Tk::Dialog;
--- Tk800.022/oddmenu_demo.wiget	Tue Jul 27 20:20:24 1999
+++ Tk800.022/oddmenu_demo	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 use Tk;
 
--- Tk800.022/pfm.wiget	Tue Jul 27 20:21:11 1999
+++ Tk800.022/pfm	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use POSIX qw(getcwd);
 
 use Tk;
--- Tk800.022/pictgrid.wiget	Tue Jul 27 20:21:11 1999
+++ Tk800.022/pictgrid	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 use Tk;
 
--- Tk800.022/pipe_test.wiget	Tue Jul 27 20:21:11 1999
+++ Tk800.022/pipe_test	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 
 require Tk::IO;
--- Tk800.022/pixmap_test.wiget	Tue Jul 27 20:21:11 1999
+++ Tk800.022/pixmap_test	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 use Tk;
 require Tk::Pixmap;
--- Tk800.022/popup.wiget	Tue Jul 27 20:21:21 1999
+++ Tk800.022/popup	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 
 use Tk;
--- Tk800.022/popup_menu.wiget	Tue Jul 27 20:21:21 1999
+++ Tk800.022/popup_menu	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 
 use Tk;
--- Tk800.022/prop_demo.wiget	Tue Jul 27 20:21:21 1999
+++ Tk800.022/prop_demo	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 
--- Tk800.022/ptked.wiget	Sat Apr 29 10:20:31 2000
+++ Tk800.022/ptked	Wed Nov 15 17:15:07 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 use IO;
 use Socket;
--- Tk800.022/ptknews.wiget	Tue Jul 27 20:21:21 1999
+++ Tk800.022/ptknews	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use strict;
 use Tk;
--- Tk800.022/scale_test.wiget	Tue Jul 27 20:21:21 1999
+++ Tk800.022/scale_test	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 
--- Tk800.022/scrolled_demo.wiget	Tue Jul 27 20:21:21 1999
+++ Tk800.022/scrolled_demo	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 use strict;
--- Tk800.022/sillymenu.wiget	Tue Jul 27 20:21:21 1999
+++ Tk800.022/sillymenu	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 use Tk;
 
--- Tk800.022/sockets.wiget	Tue Jul 27 20:21:21 1999
+++ Tk800.022/sockets	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use IO::Socket;
 use Tk::IO;
--- Tk800.022/stripblib.wiget	Tue Jul 27 20:21:21 1999
+++ Tk800.022/stripblib	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use File::Find;
 find(\&strip,'blib/arch');
 
--- Tk800.022/style_test.wiget	Tue Jul 27 20:21:21 1999
+++ Tk800.022/style_test	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 use Tk::HList;    
--- Tk800.022/submethods.wiget	Tue Jul 27 20:21:21 1999
+++ Tk800.022/submethods	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 $meth = "pTk/Methods.def";
 
 %sub = ();
--- Tk800.022/symbols.wiget	Tue Jul 27 20:21:21 1999
+++ Tk800.022/symbols	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Config;
 
--- Tk800.022/table_demo.wiget	Tue Jul 27 20:21:22 1999
+++ Tk800.022/table_demo	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 use Tk::Table;
--- Tk800.022/tainted.wiget	Tue Jul 27 20:21:22 1999
+++ Tk800.022/tainted	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -T
+#!/usr/bin/perl -T
 
 use Tk;
 
--- Tk800.022/tcl2perl.wiget	Thu Feb 17 21:51:43 2000
+++ Tk800.022/tcl2perl	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 # Highly experimental tcl -> perl converter, aimed at eventually
 # completeing Tix merge
--- Tk800.022/tcolour.wiget	Tue Jul 27 20:21:22 1999
+++ Tk800.022/tcolour	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 
--- Tk800.022/text_demo.wiget	Tue Jul 27 20:21:22 1999
+++ Tk800.022/text_demo	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 require Tk::Text;
--- Tk800.022/tie_basic.wiget	Tue Jul 27 20:21:22 1999
+++ Tk800.022/tie_basic	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl 
+#!/usr/bin/perl 
 
 # use Memory (2440);
 
--- Tk800.022/tiehandle.wiget	Tue Jul 27 20:21:22 1999
+++ Tk800.022/tiehandle	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 
--- Tk800.022/tietext.wiget	Tue Jul 27 20:21:22 1999
+++ Tk800.022/tietext	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 
 {package Tk::Text::Contents;
--- Tk800.022/time.wiget	Tue Jul 27 20:21:22 1999
+++ Tk800.022/time	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk::Stats 'Start';
 use strict;
 use Tk;
--- Tk800.022/tiny.wiget	Sat Apr  1 10:34:00 2000
+++ Tk800.022/tiny	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 use Tk;
 use Tk::widgets qw(Button);  
--- Tk800.022/tixwm_demo.wiget	Tue Jul 27 20:21:23 1999
+++ Tk800.022/tixwm_demo	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 use Tk::Dialog;
--- Tk800.022/tkperldoc.wiget	Sun Dec 19 18:28:57 1999
+++ Tk800.022/tkperldoc	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use strict;
 use Tk;
--- Tk800.022/tkpsh.wiget	Tue Jul 27 20:21:25 1999
+++ Tk800.022/tkpsh	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 require 5.001;
 use IO::Handle;
--- Tk800.022/tktail.wiget	Tue Jul 27 20:21:25 1999
+++ Tk800.022/tktail	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 use Tk;
 use IO::File;
--- Tk800.022/two_main.wiget	Sat Apr  1 09:33:59 2000
+++ Tk800.022/two_main	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 use Tk::Xrm;
--- Tk800.022/uninstall.wiget	Tue Jul 27 20:21:25 1999
+++ Tk800.022/uninstall	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 BEGIN {exec('perl5.002',$0,@ARGV)  unless ($] >= 5.002)} 
 use Config;
 use Getopt::Long;
--- Tk800.022/xlib_demo.wiget	Tue Jul 27 20:21:25 1999
+++ Tk800.022/xlib_demo	Wed Nov 15 17:10:09 2000
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 
