diff -Nru Tk-804.027.vanilla/Tk/Widget.pm Tk-804.027/Tk/Widget.pm
--- Tk-804.027.vanilla/Tk/Widget.pm	2004-03-17 17:00:32.000000000 +0100
+++ Tk-804.027/Tk/Widget.pm	2005-03-11 20:41:39.241813232 +0100
@@ -60,7 +60,7 @@
 sub True  { 1 }
 sub False { 0 }
 
-use Tk::Submethods( 'grab' =>  [qw(current status release -global)],
+use Tk::Submethods ( 'grab' =>  [qw(current status release -global)],
                     'focus' => [qw(-force -lastfor)],
                     'pack'  => [qw(configure forget info propagate slaves)],
                     'grid'  => [qw(bbox columnconfigure configure forget info location propagate rowconfigure size slaves)],
