--- Tk800.022/Tk/Widget.pm.wiget	Sat Apr 29 11:36:41 2000
+++ Tk800.022/Tk/Widget.pm	Wed Nov 15 17:20:01 2000
@@ -58,7 +58,7 @@
 sub True  { 1 }
 sub False { 0 }
 
-use Tk::Submethods( 'grab' =>  [qw(current status release -global)],
+use Tk::Submethods ( 'grab' =>  [qw(current status release -global)],
                     'focus' => [qw(-force -lastfor)],
                     'pack'  => [qw(configure forget info propagate slaves)],
                     'grid'  => [qw(bbox columnconfigure configure forget info location propagate rowconfigure size slaves)],
--- Tk800.022/Tk.pm.wiget	Sat Apr 29 11:36:41 2000
+++ Tk800.022/Tk.pm	Wed Nov 15 17:22:02 2000
@@ -9,7 +9,7 @@
 # derivation from Tk8.0 sources.
 #
 package Tk;
-require 5.00404;
+require 5.004_04;
 use     Tk::Event ();
 use     AutoLoader qw(AUTOLOAD);
 use     DynaLoader;
