--- Tk800.014.paths/Contrib/TickerTape	Sat Mar 21 18:57:22 1998
+++ Tk800.014/Contrib/TickerTape	Mon Nov  8 18:42:17 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 =head1 NAME 
 
--- Tk800.014.paths/Contrib/book-examples/f-15.5.pl	Mon Dec 29 13:11:12 1997
+++ Tk800.014/Contrib/book-examples/f-15.5.pl	Mon Nov  8 18:42:17 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 #
 # Figure 15.5, p153
--- Tk800.014.paths/Contrib/book-examples/f-16.1.pl	Mon Dec 29 13:11:12 1997
+++ Tk800.014/Contrib/book-examples/f-16.1.pl	Mon Nov  8 18:42:17 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 #
 # Figure 16.1, p159
--- Tk800.014.paths/Contrib/book-examples/f-16.10.pl	Mon Dec 29 13:11:12 1997
+++ Tk800.014/Contrib/book-examples/f-16.10.pl	Mon Nov  8 18:42:17 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 #
 # Figure 16.8, p166
--- Tk800.014.paths/Contrib/book-examples/f-16.2.pl	Mon Dec 29 13:11:12 1997
+++ Tk800.014/Contrib/book-examples/f-16.2.pl	Mon Nov  8 18:42:17 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 #
 # Figure 16.2, p161
--- Tk800.014.paths/Contrib/book-examples/f-16.3.pl	Mon Dec 29 13:11:12 1997
+++ Tk800.014/Contrib/book-examples/f-16.3.pl	Mon Nov  8 18:42:17 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 #
 # Figure 16.3, p162
--- Tk800.014.paths/Contrib/book-examples/f-16.6.pl	Mon Dec 29 13:11:12 1997
+++ Tk800.014/Contrib/book-examples/f-16.6.pl	Mon Nov  8 18:42:17 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 #
 # Figure 16.6, p165
--- Tk800.014.paths/Contrib/book-examples/f-16.7.pl	Mon Dec 29 13:11:12 1997
+++ Tk800.014/Contrib/book-examples/f-16.7.pl	Mon Nov  8 18:42:17 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 #
 # Figure 16.7, p166
--- Tk800.014.paths/Contrib/book-examples/f-16.8.pl	Mon Dec 29 13:11:12 1997
+++ Tk800.014/Contrib/book-examples/f-16.8.pl	Mon Nov  8 18:42:17 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 #
 # Figure 16.8, p166
--- Tk800.014.paths/Contrib/book-examples/f-16.9.pl	Mon Dec 29 13:11:12 1997
+++ Tk800.014/Contrib/book-examples/f-16.9.pl	Mon Nov  8 18:42:17 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 #
 # Figure 16.8, p166
--- Tk800.014.paths/Contrib/fd_demo	Mon Dec 29 13:11:12 1997
+++ Tk800.014/Contrib/fd_demo	Mon Nov  8 18:42:17 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 require Tk::FileDialog;
--- Tk800.014.paths/Contrib/testlistbox	Mon Dec 29 13:11:12 1997
+++ Tk800.014/Contrib/testlistbox	Mon Nov  8 18:42:17 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 use Tk;
 use Tk::TiedListbox;
 
--- Tk800.014.paths/DragDrop/drag_test	Tue Jun  2 20:28:30 1998
+++ Tk800.014/DragDrop/drag_test	Mon Nov  8 18:42:18 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 use strict;
 use Tk::DragDrop qw(Sun);
--- Tk800.014.paths/DragDrop/has_sun	Mon Dec 29 13:11:13 1997
+++ Tk800.014/DragDrop/has_sun	Mon Nov  8 18:42:18 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 
 my $mw = MainWindow->new;
--- Tk800.014.paths/DragDrop/local_test	Mon Dec 29 13:11:13 1997
+++ Tk800.014/DragDrop/local_test	Mon Nov  8 18:42:18 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 use strict;
 use Tk::DragDrop;
--- Tk800.014.paths/DragDrop/motion_test	Mon Dec 29 13:11:13 1997
+++ Tk800.014/DragDrop/motion_test	Mon Nov  8 18:42:18 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 use strict;
 use Tk::DragDrop;
--- Tk800.014.paths/DragDrop/selection	Sat Feb 28 23:15:26 1998
+++ Tk800.014/DragDrop/selection	Mon Nov  8 18:42:18 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 use strict;
 
--- Tk800.014.paths/DragDrop/site_test	Mon Sep 28 19:40:26 1998
+++ Tk800.014/DragDrop/site_test	Mon Nov  8 18:42:18 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 use strict;
 use Tk::DropSite;
--- Tk800.014.paths/Extensions/ImageBack/background	Fri Feb 13 18:38:47 1998
+++ Tk800.014/Extensions/ImageBack/background	Mon Nov  8 18:42:18 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 use Tk::ImageBack;
 use Tk::Photo;
--- Tk800.014.paths/HList/Tix2perl	Mon Dec 29 13:11:13 1997
+++ Tk800.014/HList/Tix2perl	Mon Nov  8 18:42:18 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk::Pretty;
 
--- Tk800.014.paths/TixGrid/tgdemo_config	Tue Jun 30 19:49:16 1998
+++ Tk800.014/TixGrid/tgdemo_config	Mon Nov  8 18:42:18 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 # tixgrid demo
 #	standalone or sourceable in ptksh
 
--- Tk800.014.paths/WinPhoto/try	Sat Nov  7 19:00:47 1998
+++ Tk800.014/WinPhoto/try	Mon Nov  8 18:42:20 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;             
 use Tk::WinPhoto;
 use Cwd;
--- Tk800.014.paths/Xlib/drops	Sat Jul 11 12:30:29 1998
+++ Tk800.014/Xlib/drops	Mon Nov  8 18:42:20 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 
 use Tk;
--- Tk800.014.paths/Xlib/load	Mon Dec 29 13:11:23 1997
+++ Tk800.014/Xlib/load	Mon Nov  8 18:42:20 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 use Tk;
 use Tk::Xlib;
--- Tk800.014.paths/Xlib/motif_site	Mon Dec 29 13:11:23 1997
+++ Tk800.014/Xlib/motif_site	Mon Nov  8 18:42:20 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 use Tk::Xlib;
 use Data::Dumper;
--- Tk800.014.paths/Xlib/tree_demo	Mon Dec 29 13:11:23 1997
+++ Tk800.014/Xlib/tree_demo	Mon Nov  8 18:42:20 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 use Tk::Xlib;
 use Tk::HList;
--- Tk800.014.paths/add_version	Sat Jul 11 12:32:47 1998
+++ Tk800.014/add_version	Mon Nov  8 18:42:20 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 use File::Find;
 use Getopt::Std;
--- Tk800.014.paths/adj_demo	Sat Feb 28 23:16:15 1998
+++ Tk800.014/adj_demo	Mon Nov  8 18:42:20 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl  -w
+#!/usr/bin/perl  -w
 use Tk;
 require Tk::TextUndo;
 
--- Tk800.014.paths/after_demo	Fri Feb  6 19:37:49 1998
+++ Tk800.014/after_demo	Mon Nov  8 18:42:20 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 use strict;
--- Tk800.014.paths/after_leak	Fri Feb 13 18:38:49 1998
+++ Tk800.014/after_leak	Mon Nov  8 18:42:20 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 use English;
--- Tk800.014.paths/animate	Sun Feb 15 13:50:02 1998
+++ Tk800.014/animate	Mon Nov  8 18:42:20 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 
 my $mw = MainWindow->new;
--- Tk800.014.paths/autoraise	Mon Dec 29 13:11:24 1997
+++ Tk800.014/autoraise	Mon Nov  8 18:42:20 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 
--- Tk800.014.paths/basic_demo	Fri Nov 13 19:26:10 1998
+++ Tk800.014/basic_demo	Mon Nov  8 18:42:20 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl 
+#!/usr/bin/perl 
 
 # use Memory (2440);
 
--- Tk800.014.paths/bin/defs	Mon Dec 29 13:11:24 1997
+++ Tk800.014/bin/defs	Mon Nov  8 18:42:20 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl 
+#!/usr/bin/perl 
 
 %def   = ();
 %need  = ();
--- Tk800.014.paths/bin/defunc	Mon Dec 29 13:11:24 1997
+++ Tk800.014/bin/defunc	Mon Nov  8 18:42:20 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 while (<>)
  {
--- Tk800.014.paths/bin/newstyle	Mon Dec 29 13:11:24 1997
+++ Tk800.014/bin/newstyle	Mon Nov  8 18:42:20 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 @files = @ARGV;
 
--- Tk800.014.paths/bin/perms	Mon Dec 29 13:11:24 1997
+++ Tk800.014/bin/perms	Mon Nov  8 18:42:20 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 use File::Find;
 
--- Tk800.014.paths/bin/unres	Mon Dec 29 13:11:24 1997
+++ Tk800.014/bin/unres	Mon Nov  8 18:42:20 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 %done = ();
 %defs = ();
--- Tk800.014.paths/bin/uselib	Mon Dec 29 13:11:24 1997
+++ Tk800.014/bin/uselib	Mon Nov  8 18:42:20 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl 
+#!/usr/bin/perl 
 
 use File::Find;
 
--- Tk800.014.paths/bindtest	Mon Dec 29 13:11:24 1997
+++ Tk800.014/bindtest	Mon Nov  8 18:42:20 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 use Tk::Menubar;
 
--- Tk800.014.paths/build	Fri Jan 22 18:44:50 1999
+++ Tk800.014/build	Mon Nov  8 18:42:20 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Config;
 my $make = $Config{'make'};
 my @para;
--- Tk800.014.paths/bulkedit	Sat Jul 11 12:32:48 1998
+++ Tk800.014/bulkedit	Mon Nov  8 18:42:20 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 use Tk;
 use Tk::LabEntry;
--- Tk800.014.paths/canvas_ps	Mon Dec 29 13:11:24 1997
+++ Tk800.014/canvas_ps	Mon Nov  8 18:42:20 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 
--- Tk800.014.paths/canvas_scroll	Sun Feb  8 10:18:10 1998
+++ Tk800.014/canvas_scroll	Mon Nov  8 18:42:20 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 
--- Tk800.014.paths/cbtest	Mon Dec 29 13:11:24 1997
+++ Tk800.014/cbtest	Mon Nov  8 18:42:20 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 use Tk;
 use Tk::Pretty;
--- Tk800.014.paths/check_syms	Mon Apr  5 13:10:07 1999
+++ Tk800.014/check_syms	Mon Nov  8 18:42:20 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use File::Find;     
 use Config; 
 my @libs;
--- Tk800.014.paths/checklen	Mon Dec 29 13:11:24 1997
+++ Tk800.014/checklen	Mon Nov  8 18:42:20 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use ExtUtils::Manifest "/mani/";
 
--- Tk800.014.paths/classtree	Thu Aug  6 20:28:57 1998
+++ Tk800.014/classtree	Mon Nov  8 18:42:20 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 use Tk;    
 use Tk::widgets qw(Tree Menubar);
--- Tk800.014.paths/cleanup	Sat May  9 14:51:37 1998
+++ Tk800.014/cleanup	Mon Nov  8 18:42:20 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use File::Find;
 find(\&wanted,'.');
 
--- Tk800.014.paths/clip_bug	Mon Dec 29 13:11:24 1997
+++ Tk800.014/clip_bug	Mon Nov  8 18:42:20 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 # Show a Tk canvas bug:
 # when scrolling a canvas, clipping for embedded windows
 # uses the embedded windows parent, not the canvas.
--- Tk800.014.paths/compile/findloader	Fri Mar 12 18:43:06 1999
+++ Tk800.014/compile/findloader	Mon Nov  8 18:42:20 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Config; 
 my $sfx = $Config{'lib_ext'};
 foreach my $dir (@INC)
--- Tk800.014.paths/compile/splitfunc	Sat Nov 21 23:02:28 1998
+++ Tk800.014/compile/splitfunc	Mon Nov  8 18:42:20 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use IO::File;
 
 while (<>)
--- Tk800.014.paths/copydate	Sat Nov 14 11:45:34 1998
+++ Tk800.014/copydate	Mon Nov  8 18:42:20 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl 
+#!/usr/bin/perl 
 
 use File::Find;
 use Time::Local;
--- Tk800.014.paths/cursor_demo	Sat Feb 28 23:16:58 1998
+++ Tk800.014/cursor_demo	Mon Nov  8 18:42:20 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 
 my $mw = MainWindow->new;
--- Tk800.014.paths/demos/browse	Fri Feb  6 14:44:57 1998
+++ Tk800.014/demos/browse	Mon Nov  8 18:42:21 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 #
 # This script generates a directory browser, which lists the working directory and allows you to open files or subdirectories
 # by double-clicking.
--- Tk800.014.paths/demos/color_editor	Fri Feb  6 14:44:59 1998
+++ Tk800.014/demos/color_editor	Mon Nov  8 18:42:21 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 #
 # ColorEditor demonstration.
 
--- Tk800.014.paths/demos/composite	Fri Feb  6 14:45:00 1998
+++ Tk800.014/demos/composite	Mon Nov  8 18:42:21 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 
 use English;
--- Tk800.014.paths/demos/dialog	Mon Dec 29 13:11:27 1997
+++ Tk800.014/demos/dialog	Mon Nov  8 18:42:22 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 # Demonstrate how to use Dialog to create and manipulate dialog objects.
 # lusol@Lehigh.EDU  95/06/14
--- Tk800.014.paths/demos/hello	Mon Dec 29 13:11:27 1997
+++ Tk800.014/demos/hello	Mon Nov  8 18:42:22 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 #
 # Simple Tk script to create a button that prints "Hello, world".  Click on 
 # the button to terminate the program.
--- Tk800.014.paths/demos/ixset	Fri Feb  6 14:45:45 1998
+++ Tk800.014/demos/ixset	Mon Nov  8 18:42:22 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 # A nice interface to "xset" to change X server settings
 #
--- Tk800.014.paths/demos/rmt	Fri Mar 19 10:41:04 1999
+++ Tk800.014/demos/rmt	Mon Nov  8 18:42:22 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -wT
+#!/usr/bin/perl -wT
 #
 # This script implements a simple remote-control mechanism for Tk 
 # applications.  It allows you to select an application and then type 
--- Tk800.014.paths/demos/rolodex	Fri Feb  6 14:45:47 1998
+++ Tk800.014/demos/rolodex	Mon Nov  8 18:42:22 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 #
 # This script was written as an entry in Tom LaStrange's rolodex benchmark. 
 # It creates something that has some of the look and feel of a rolodex program,# although it's lifeless and doesn't actually do the rolodex application.
--- Tk800.014.paths/demos/timer	Mon Dec 29 13:11:27 1997
+++ Tk800.014/demos/timer	Mon Nov  8 18:42:22 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 #
 # This script generates a counter with start and stop buttons.  Exit with 
 # Ctrl/c or Ctrl/q.
--- Tk800.014.paths/demos/tom	Mon Dec 29 13:11:27 1997
+++ Tk800.014/demos/tom	Mon Nov  8 18:42:22 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 #
 # Sample program to get Tom Christiansen going with Perl/Tk.
 
--- Tk800.014.paths/demos/widget	Sat Sep  5 18:40:22 1998
+++ Tk800.014/demos/widget	Mon Nov  8 18:42:22 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 eval 'exec /usr/local/bin/perl -w -S $0 ${1+"$@"}'
     if 0; # not running under some shell
--- Tk800.014.paths/demos/widget_old	Fri Feb  6 14:45:48 1998
+++ Tk800.014/demos/widget_old	Mon Nov  8 18:42:22 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 #
 # This script demonstrates the various widgets provided by Tk, along with many of the features of the Tk toolkit.  This file
 # only contains code to generate the main window for the application, which invokes individual demonstrations.  The code for 
--- Tk800.014.paths/derived_test	Sat Jul 11 12:30:29 1998
+++ Tk800.014/derived_test	Mon Nov  8 18:42:22 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 require Tk::Text;
 
--- Tk800.014.paths/destroy_test	Mon Dec 29 13:11:27 1997
+++ Tk800.014/destroy_test	Mon Nov  8 18:42:22 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 
 use Tk;
--- Tk800.014.paths/dfpy	Mon Dec 29 13:11:27 1997
+++ Tk800.014/dfpy	Mon Nov  8 18:42:22 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 use ExtUtils::Manifest "/mani/";
 use Cwd;
--- Tk800.014.paths/disbutton	Sat Jul 11 12:30:29 1998
+++ Tk800.014/disbutton	Mon Nov  8 18:42:22 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 package StickyButton;
 require Tk::Widget;
--- Tk800.014.paths/dnd_demo	Thu May 14 23:01:40 1998
+++ Tk800.014/dnd_demo	Mon Nov  8 18:42:22 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 #
 # Demo code to exhibit features of dynamic construction and destruction of 
 # local (within app.) Drag and Drop sources and sinks. Written for and tested 
--- Tk800.014.paths/dump_test	Sat May 16 20:58:54 1998
+++ Tk800.014/dump_test	Mon Nov  8 18:42:22 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 my $top = MainWindow->new;
 my $i   = $top->Photo(-file => Tk->findINC('icon.gif'));
--- Tk800.014.paths/embed	Sat Feb 28 19:27:37 1998
+++ Tk800.014/embed	Mon Nov  8 18:42:22 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w 
+#!/usr/bin/perl -w 
 
 use Tk;
 use IO::File;
--- Tk800.014.paths/error_prop	Mon Dec 29 13:11:30 1997
+++ Tk800.014/error_prop	Mon Nov  8 18:42:22 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 use Tk::ErrorDialog;
--- Tk800.014.paths/ewidth_demo	Mon Dec 29 13:11:31 1997
+++ Tk800.014/ewidth_demo	Mon Nov  8 18:42:22 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 use strict;
 
--- Tk800.014.paths/findbase	Mon Dec 29 13:11:31 1997
+++ Tk800.014/findbase	Mon Nov  8 18:42:22 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use File::Find;
 
--- Tk800.014.paths/floppy	Mon Dec 29 13:11:31 1997
+++ Tk800.014/floppy	Mon Nov  8 18:42:22 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 use ExtUtils::Manifest "/mani/";
 use Cwd;
--- Tk800.014.paths/font_demo	Mon Dec 29 13:11:31 1997
+++ Tk800.014/font_demo	Mon Nov  8 18:42:22 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 use Tk::Font;
 use Tk::ROText;
--- Tk800.014.paths/font_test	Mon Dec 29 13:11:31 1997
+++ Tk800.014/font_test	Mon Nov  8 18:42:22 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 
 
--- Tk800.014.paths/fonts	Thu Nov  5 23:32:12 1998
+++ Tk800.014/fonts	Mon Nov  8 18:42:22 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 use Tk;              
 use Data::Dumper;
--- Tk800.014.paths/gdb.plx	Sat Jun 20 12:25:13 1998
+++ Tk800.014/gdb.plx	Mon Nov  8 18:42:22 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Config;
 open(INIT,">.gdbinit") || die;
 my $perl = $^X;
--- Tk800.014.paths/geo_mgr	Sat Jul 11 12:30:29 1998
+++ Tk800.014/geo_mgr	Mon Nov  8 18:42:22 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 
--- Tk800.014.paths/geom	Mon Dec 29 13:11:31 1997
+++ Tk800.014/geom	Mon Nov  8 18:42:22 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 
--- Tk800.014.paths/grepc	Mon Dec 29 13:11:31 1997
+++ Tk800.014/grepc	Mon Nov  8 18:42:22 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl 
+#!/usr/bin/perl 
 
 use File::Find;
 use Getopt::Std;
--- Tk800.014.paths/grepperl	Tue Oct 13 13:33:47 1998
+++ Tk800.014/grepperl	Mon Nov  8 18:42:22 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl 
+#!/usr/bin/perl 
 
 use Getopt::Std;
 
--- Tk800.014.paths/grid_adj	Sat Feb 28 23:17:52 1998
+++ Tk800.014/grid_adj	Mon Nov  8 18:42:22 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl  -w
+#!/usr/bin/perl  -w
 use Tk;
 require Tk::TextUndo;
 require Tk::Adjuster;
--- Tk800.014.paths/grid_test	Mon Dec 29 13:11:31 1997
+++ Tk800.014/grid_test	Mon Nov  8 18:42:22 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 my $mw = MainWindow->new;
 open(FILE,__FILE__);
--- Tk800.014.paths/gridbug	Mon Dec 29 13:11:31 1997
+++ Tk800.014/gridbug	Mon Nov  8 18:42:22 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 use Carp;
 $top = MainWindow->new();
--- Tk800.014.paths/hackMM	Mon Dec 29 13:11:31 1997
+++ Tk800.014/hackMM	Mon Nov  8 18:42:22 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use File::Find;
 
--- Tk800.014.paths/hlfm	Mon Dec 29 13:11:31 1997
+++ Tk800.014/hlfm	Mon Nov  8 18:42:22 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 use Tk;
 require Tk::Dirlist;
--- Tk800.014.paths/hlist_bg	Mon Dec 29 13:11:31 1997
+++ Tk800.014/hlist_bg	Mon Nov  8 18:42:22 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 require Tk::HList;
 
--- Tk800.014.paths/iconwin	Fri Jun 26 16:29:39 1998
+++ Tk800.014/iconwin	Mon Nov  8 18:42:22 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 
--- Tk800.014.paths/image_test	Tue Feb 10 18:47:47 1998
+++ Tk800.014/image_test	Mon Nov  8 18:42:33 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 
--- Tk800.014.paths/inline_bitmap	Fri Feb 13 18:38:49 1998
+++ Tk800.014/inline_bitmap	Mon Nov  8 18:42:33 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 use Tk;             
 
--- Tk800.014.paths/keyWords	Mon Dec 29 13:11:31 1997
+++ Tk800.014/keyWords	Mon Nov  8 18:42:33 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use File::Find;
 
--- Tk800.014.paths/labelpack	Mon Dec 29 13:11:31 1997
+++ Tk800.014/labelpack	Mon Nov  8 18:42:33 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 require Tk::LabEntry;
--- Tk800.014.paths/lbformat_demo	Mon Dec 29 13:11:31 1997
+++ Tk800.014/lbformat_demo	Mon Nov  8 18:42:34 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 use Tk;
 use Tk::Font;
--- Tk800.014.paths/leak_test	Mon Dec 29 13:11:31 1997
+++ Tk800.014/leak_test	Mon Nov  8 18:42:34 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 use Tk::widgets qw(Button);
--- Tk800.014.paths/lentry	Sat Jul 11 12:30:29 1998
+++ Tk800.014/lentry	Mon Nov  8 18:42:34 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 
--- Tk800.014.paths/local_demo	Mon Dec 29 13:11:31 1997
+++ Tk800.014/local_demo	Mon Nov  8 18:42:34 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 use Tk qw(MainLoop *widget);
 use Carp;
--- Tk800.014.paths/ls_table	Mon Dec 29 13:11:31 1997
+++ Tk800.014/ls_table	Mon Nov  8 18:42:34 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 use Tk::Table;
--- Tk800.014.paths/lsfonts	Mon Dec 29 13:11:31 1997
+++ Tk800.014/lsfonts	Mon Nov  8 18:42:34 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 use Tk::Xlib;
--- Tk800.014.paths/menubar	Mon Dec 29 13:11:31 1997
+++ Tk800.014/menubar	Mon Nov  8 18:42:34 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 
--- Tk800.014.paths/menubug	Mon Jun 29 20:21:35 1998
+++ Tk800.014/menubug	Mon Nov  8 18:42:34 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 use strict;
 
--- Tk800.014.paths/mitem_test	Sat Jul 11 12:30:29 1998
+++ Tk800.014/mitem_test	Mon Nov  8 18:42:34 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
  
 require 5.003;
 use English;
--- Tk800.014.paths/mkExt	Mon Dec 29 13:11:31 1997
+++ Tk800.014/mkExt	Mon Nov  8 18:42:34 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 sub translate
 {
--- Tk800.014.paths/mkdist	Wed Jun 10 19:41:16 1998
+++ Tk800.014/mkdist	Mon Nov  8 18:42:34 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 
 use ExtUtils::Manifest "/mani/";
--- Tk800.014.paths/motif_hack	Mon Dec 29 13:11:31 1997
+++ Tk800.014/motif_hack	Mon Nov  8 18:42:34 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Data::Dumper;
 use strict;
 use Tk;
--- Tk800.014.paths/nested_dialog	Mon Dec 29 13:11:31 1997
+++ Tk800.014/nested_dialog	Mon Nov  8 18:42:34 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 use Tk::Dialog;
--- Tk800.014.paths/oddmenu_demo	Mon Dec 29 13:11:31 1997
+++ Tk800.014/oddmenu_demo	Mon Nov  8 18:42:34 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 use Tk;
 
--- Tk800.014.paths/pTk/LangIO.h.PL	Thu Mar  5 20:51:19 1998
+++ Tk800.014/pTk/LangIO.h.PL	Mon Nov  8 18:42:23 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Config;
 my @gard = qw(
 STDIO_H
--- Tk800.014.paths/pTk/Tcl-pTk	Wed Apr  8 21:07:13 1998
+++ Tk800.014/pTk/Tcl-pTk	Mon Nov  8 18:42:23 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Carp;
 
--- Tk800.014.paths/pTk/chext	Mon Dec 29 13:11:32 1997
+++ Tk800.014/pTk/chext	Mon Nov  8 18:42:23 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 open(NEED,"<need") || die;
 while (<NEED>)
--- Tk800.014.paths/pTk/counts	Mon Dec 29 13:11:32 1997
+++ Tk800.014/pTk/counts	Mon Nov  8 18:42:23 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 sub count
 {
--- Tk800.014.paths/pTk/findX	Mon Dec 29 13:11:32 1997
+++ Tk800.014/pTk/findX	Mon Nov  8 18:42:23 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 BEGIN { require Exporter };
 
--- Tk800.014.paths/pTk/findcore	Mon Dec 29 13:11:32 1997
+++ Tk800.014/pTk/findcore	Mon Nov  8 18:42:23 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 BEGIN { require Exporter };
 
--- Tk800.014.paths/pTk/findstrings	Mon Dec 29 13:11:32 1997
+++ Tk800.014/pTk/findstrings	Mon Nov  8 18:42:23 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 
 %table = ();
--- Tk800.014.paths/pTk/from	Mon Dec 29 13:11:32 1997
+++ Tk800.014/pTk/from	Mon Nov  8 18:42:23 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 
 use File::Find;
--- Tk800.014.paths/pTk/genexc	Mon Dec 29 13:11:32 1997
+++ Tk800.014/pTk/genexc	Mon Nov  8 18:42:23 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 my %mTk;
 my $dir;
--- Tk800.014.paths/pTk/mTk/tclmerge	Mon Feb  9 22:48:47 1998
+++ Tk800.014/pTk/mTk/tclmerge	Mon Nov  8 18:42:28 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use strict;
 
--- Tk800.014.paths/pTk/mTk/tixmerge	Mon Feb  9 22:48:55 1998
+++ Tk800.014/pTk/mTk/tixmerge	Mon Nov  8 18:42:30 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use strict;
 
--- Tk800.014.paths/pTk/makeenglish	Sat Jul 11 12:30:30 1998
+++ Tk800.014/pTk/makeenglish	Mon Nov  8 18:42:33 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 # BEGIN { push(@INC, qw(../../../lib ../../lib ../lib lib)) }
 
 die "Must be executed from Tk/pTk distribution directory\n" unless -f "Lang.h";
--- Tk800.014.paths/pTk/mkVFunc	Fri May  8 18:05:39 1998
+++ Tk800.014/pTk/mkVFunc	Mon Nov  8 18:42:33 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Getopt::Std;
 my %opt;
--- Tk800.014.paths/pTk/mkXinfo	Mon Dec 29 13:11:50 1997
+++ Tk800.014/pTk/mkXinfo	Mon Nov  8 18:42:33 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 $file = shift;
 
--- Tk800.014.paths/pTk/mkexc	Mon Dec 29 13:11:50 1997
+++ Tk800.014/pTk/mkexc	Mon Nov  8 18:42:33 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 use vars qw(%ref %def);
 require "process_object";
--- Tk800.014.paths/pTk/mkmethods	Mon Dec 29 13:11:50 1997
+++ Tk800.014/pTk/mkmethods	Mon Nov  8 18:42:33 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 $dst = shift;
 
--- Tk800.014.paths/pTk/tdiff	Sun Feb  8 12:30:59 1998
+++ Tk800.014/pTk/tdiff	Mon Nov  8 18:42:33 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 sub fail
 {
--- Tk800.014.paths/pTk/why	Thu Feb  5 16:37:13 1998
+++ Tk800.014/pTk/why	Mon Nov  8 18:42:33 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use strict;        
 
--- Tk800.014.paths/pfm	Sat Jul 11 12:30:30 1998
+++ Tk800.014/pfm	Mon Nov  8 18:42:34 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use POSIX qw(getcwd);
 
 use Tk;
--- Tk800.014.paths/pictgrid	Mon Dec 29 13:11:52 1997
+++ Tk800.014/pictgrid	Mon Nov  8 18:42:34 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 use Tk;
 
--- Tk800.014.paths/pipe_test	Mon Dec 29 13:11:52 1997
+++ Tk800.014/pipe_test	Mon Nov  8 18:42:34 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 
 require Tk::IO;
--- Tk800.014.paths/pixmap_test	Mon Dec 29 13:11:52 1997
+++ Tk800.014/pixmap_test	Mon Nov  8 18:42:34 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 use Tk;
 require Tk::Pixmap;
--- Tk800.014.paths/pod/extract/extpod	Sat Apr 18 12:00:20 1998
+++ Tk800.014/pod/extract/extpod	Mon Nov  8 18:42:35 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;            
 use Getopt::Std;
 use ExtUtils::Manifest "/mani/";
--- Tk800.014.paths/pod/recat	Sun Mar 22 13:54:46 1998
+++ Tk800.014/pod/recat	Mon Nov  8 18:42:36 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 my $sec;            
 my %cat;       
--- Tk800.014.paths/pod/subs	Mon Mar 23 19:07:41 1998
+++ Tk800.014/pod/subs	Mon Nov  8 18:42:37 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use blib;
 use Tk;                     
 use strict;
--- Tk800.014.paths/popup	Mon Dec 29 13:11:52 1997
+++ Tk800.014/popup	Mon Nov  8 18:42:37 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 
 use Tk;
--- Tk800.014.paths/popup_menu	Mon Dec 29 13:11:52 1997
+++ Tk800.014/popup_menu	Mon Nov  8 18:42:37 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 
 use Tk;
--- Tk800.014.paths/prop_demo	Mon Dec 29 13:11:52 1997
+++ Tk800.014/prop_demo	Mon Nov  8 18:42:37 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 
--- Tk800.014.paths/ptked	Thu Dec 10 21:54:48 1998
+++ Tk800.014/ptked	Mon Nov  8 18:42:37 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 use Tk::DropSite qw(Sun);
--- Tk800.014.paths/ptknews	Sat Feb 28 23:18:15 1998
+++ Tk800.014/ptknews	Mon Nov  8 18:42:37 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use strict;
 use Tk;
--- Tk800.014.paths/scale_test	Mon Dec 29 13:11:52 1997
+++ Tk800.014/scale_test	Mon Nov  8 18:42:37 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 
--- Tk800.014.paths/scrolled_demo	Mon Dec 29 13:11:52 1997
+++ Tk800.014/scrolled_demo	Mon Nov  8 18:42:37 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 use strict;
--- Tk800.014.paths/sillymenu	Fri Nov 13 19:47:28 1998
+++ Tk800.014/sillymenu	Mon Nov  8 18:42:37 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 use Tk;
 
--- Tk800.014.paths/sockets	Mon Dec 29 13:11:52 1997
+++ Tk800.014/sockets	Mon Nov  8 18:42:37 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use IO::Socket;
 use Tk::IO;
--- Tk800.014.paths/stripblib	Thu Jul  2 20:32:40 1998
+++ Tk800.014/stripblib	Mon Nov  8 18:42:37 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use File::Find;
 find(\&strip,'blib/arch');
 
--- Tk800.014.paths/style_test	Sat May  9 12:31:49 1998
+++ Tk800.014/style_test	Mon Nov  8 18:42:37 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 use Tk::HList;    
--- Tk800.014.paths/submethods	Mon Dec 29 13:11:52 1997
+++ Tk800.014/submethods	Mon Nov  8 18:42:37 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 $meth = "pTk/Methods.def";
 
 %sub = ();
--- Tk800.014.paths/symbols	Mon Dec 29 13:11:52 1997
+++ Tk800.014/symbols	Mon Nov  8 18:42:37 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Config;
 
--- Tk800.014.paths/table_demo	Fri Feb 13 18:38:52 1998
+++ Tk800.014/table_demo	Mon Nov  8 18:42:37 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 use Tk::Table;
--- Tk800.014.paths/tainted	Mon Dec 29 13:11:52 1997
+++ Tk800.014/tainted	Mon Nov  8 18:42:37 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -T
+#!/usr/bin/perl -T
 
 use Tk;
 
--- Tk800.014.paths/tcl2perl	Mon Dec 29 13:11:52 1997
+++ Tk800.014/tcl2perl	Mon Nov  8 18:42:37 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 # Highly experimental tcl -> perl converter, aimed at eventually
 # completeing Tix merge
--- Tk800.014.paths/tcolour	Mon Dec 29 13:11:52 1997
+++ Tk800.014/tcolour	Mon Nov  8 18:42:37 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 
--- Tk800.014.paths/text_demo	Mon Dec 29 13:11:52 1997
+++ Tk800.014/text_demo	Mon Nov  8 18:42:37 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 require Tk::Text;
--- Tk800.014.paths/tie_basic	Mon Dec 29 13:11:52 1997
+++ Tk800.014/tie_basic	Mon Nov  8 18:42:37 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl 
+#!/usr/bin/perl 
 
 # use Memory (2440);
 
--- Tk800.014.paths/tiehandle	Mon Dec 29 13:11:52 1997
+++ Tk800.014/tiehandle	Mon Nov  8 18:42:37 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 
--- Tk800.014.paths/tietext	Mon Dec 29 13:11:52 1997
+++ Tk800.014/tietext	Mon Nov  8 18:42:37 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk;
 
 {package Tk::Text::Contents;
--- Tk800.014.paths/time	Mon Dec 29 13:11:53 1997
+++ Tk800.014/time	Mon Nov  8 18:42:37 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use Tk::Stats 'Start';
 use strict;
 use Tk;
--- Tk800.014.paths/tiny	Tue Mar  2 21:46:57 1999
+++ Tk800.014/tiny	Mon Nov  8 18:42:37 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 use Tk;
 use Tk::widgets qw(Button);  
--- Tk800.014.paths/tixwm_demo	Mon Dec 29 13:11:53 1997
+++ Tk800.014/tixwm_demo	Mon Nov  8 18:42:37 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 use Tk::Dialog;
--- Tk800.014.paths/tkperldoc	Mon Dec 29 13:11:53 1997
+++ Tk800.014/tkperldoc	Mon Nov  8 18:42:38 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use strict;
 use Tk;
--- Tk800.014.paths/tkpsh	Sun May 17 13:27:47 1998
+++ Tk800.014/tkpsh	Mon Nov  8 18:42:38 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 require 5.001;
 use IO::Handle;
--- Tk800.014.paths/tktail	Mon Dec 29 13:11:53 1997
+++ Tk800.014/tktail	Mon Nov  8 18:42:38 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 use strict;
 use Tk;
 use IO::File;
--- Tk800.014.paths/two_main	Mon Dec 29 13:11:53 1997
+++ Tk800.014/two_main	Mon Nov  8 18:42:38 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 
--- Tk800.014.paths/uninstall	Mon Dec 29 13:11:53 1997
+++ Tk800.014/uninstall	Mon Nov  8 18:42:38 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 BEGIN {exec('perl5.002',$0,@ARGV)  unless ($] >= 5.002)} 
 use Config;
 use Getopt::Long;
--- Tk800.014.paths/xlib_demo	Mon Dec 29 13:11:53 1997
+++ Tk800.014/xlib_demo	Mon Nov  8 18:42:38 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 use Tk;
 
